contexts/v2/index.json

```diff
@@ -0,0 +1,80 @@
+{
+  "@context": {
+    "id": "@id",
```

Comment:
This context should use `"@protected": true` in all `"@context"` blocks as a security measure. `"@protected": true` prevents unsafe term redefinition.

Reply:
Does the data integrity specification require this?

Reply:
@OR13, if it doesn't spell it out today then it will as soon as someone is able to add it.